docs/start/api.query.other.md

```diff
@@ -0,0 +1,75 @@
+# Query extras
+
+In previous sections we took a walk through queries, showing how to use one-shot queries, how to subscribe to results and how to combine multiple queries into one. This section will aim to extend that knowledge showing some other features and utilities that are available on the `api.query` interfaces.
+
+## State at a specific block
+
+Quite often is is useful (taking pruning into account, more on this later) to retrieve the state at a specific block. For instance we may wish to retrieve the current balance as well as the balance at a previous block for a specific account -
+
+```js
+...
+
+// retrieve the current block header
+const lastHdr = await api.rpc.chain.getHeader();
+
+// retrieve the balance at both at the current hash and the parentHash
+const [balanceNow, balancePrev] = await Promise.all([
+  api.query.balances.freeBalance.at(lastHdr.hash, ADDR),
+  api.query.balances.freeBalance.at(lastHdr.parentHash, ADDR)
+]);
+
+// display the difference
+console.log(`The delta was ${balanceNow.sub(balancePrev)}`);
+```
+
+In the above example, we introduce the `.at(<hash>, [params])` query. For all `.at` queries, the first parameter is always the block hash at which we want to make the query, in our example we use both the last retrieved block and the parent thereof. The params are optional as per the type of query made, for instance to retrieve the timestamp for a previous block, it would be -
+
+```js
+...
+
+// retrieve the timestampt for the  previous block
+const momentPrev = await api.query.timestamp.now.at(lastHdr.parentHash);
+```
+
+The `.at` queries are all single-shot, i.e. there are no subscription option to these, since the state for a previous block should be static. (This is true to a certain extent, i.e. when blocks have been finalized).
```

Comment:
Is there any way to listen for re-orgs?

Reply:
with `newHeads`, that is how this is constructed - https://polkadot.js.org/apps/#/explorer/forks